index.html

```diff
@@ -0,0 +1,183 @@
+        );
+        string_number = string_number.replaceAll(third_random_number, "");
+
+        random_3_num =
+          first_random_number + second_random_number + third_random_number;
+        alert(random_3_num);
+      }
+
+      //ex)[1,0,0]-> first number strike
+      //ex)[0,1,0]-> second number strike
+      //ex)[1,1,1]-> first,second, third number strike
+      function strike_check_function(random, input) {
+        var strike = [];
+        if (random[0] == input[0]) {
+          strike[0] = 1;
+        } else {
+          strike[0] = 0;
+        }
+        if (random[1] == input[1]) {
+          strike[1] = 1;
+        } else {
+          strike[1] = 0;
+        }
+        if (random[2] == input[2]) {
+          strike[2] = 1;
+        } else {
+          strike[2] = 0;
+        }
+        //alert(strike);
+        return strike;
+      }
+
+      //ex)[1,0,0]-> first number ball
+      //ex)[0,1,0]-> second number ball
+      //ex) [1,1,0] : first, second number ball
+      function ball_check_function(random, input) {
+        var ball = [];
+        if (random[0] == input[1] || random[0] == input[2]) {
+          ball[0] = 1;
+        } else {
+          ball[0] = 0;
+        }
+        if (random[1] == input[0] || random[1] == input[2]) {
+          ball[1] = 1;
+        } else {
+          ball[1] = 0;
+        }
+        if (random[2] == input[0] || random[2] == input[1]) {
+          ball[2] = 1;
+        } else {
+          ball[2] = 0;
+        }
+        //alert(ball);
+        return ball;
+      }
+
+      //check input whether it has 3-digit string number.
+      function input_enter() {
+        if (window.event.keyCode == 13) {
+          var input_value = document.getElementById("input_01").value;
+
+          if (input_value.length == 3) {
+            //check strike position [_,_,_]
+            var strike_position = strike_check_function(
+              random_3_num,
+              input_value
+            );
+            //alert(strike_position);
+
+            //check ball position [_,_,_]
+            var ball_position = ball_check_function(random_3_num, input_value);
+            //alert(strike_position + "\n" + ball_position);
+
+            //show strike position & ball position && status[win or over or count]
+            document.getElementById("text1").innerHTML =
+              "strike_position : " + strike_position;
+            document.getElementById("text2").innerHTML =
+              "ball_position : " + ball_position;
+            if (count >= 10) {
+              document.getElementById("text3").innerHTML = "GameOver ";
+            } else if (
+              strike_position[0] == 1 &&
+              strike_position[1] == 1 &&
+              strike_position[2] == 1
+            ) {
+              document.getElementById("text3").innerHTML = "Win!!! ";
+              count = 10;
+            } else {
+              document.getElementById("text3").innerHTML = "Count : " + ++count;
```

Comment:
```suggestion
              count++;
              document.getElementById("text3").innerHTML = "Count : " + count;
```
innerHTML을 하면서 count를 증가시키는 것 보다, 분리하는 것이 더 이해하기 쉬울 것 같아요.